### PR TITLE
Fix several comments syntaxes

### PR DESCRIPTION
**Pull request by Alkarex**
Mainly wrong `@return` types in comments

### CODE REVIEW

File: app/Models/StatsDAO.php [hunk 1]
```diff
@@ -238,9 +237,7 @@ protected function initStatsArray($min, $max) {
 
 	/**
 	 * Calculates feed count per category.
-	 * Returns the result as a JSON object.
-	 *
-	 * @return JSON object
+	 * @return array
```
Comment (Frenzie) [inline app/Models/StatsDAO.php:240]: There's no JSON involved at all?

**Review by Frenzie — APPROVED** [review]
Thanks!

File: app/Models/StatsDAO.php (hunk 1 above)
Comment (Alkarex) [reply app/Models/StatsDAO.php:240]: No, it is just an array

File: app/Models/Share.php [hunk 2]
```diff
@@ -51,7 +51,7 @@ public static function load($filename) {
 
 	/**
 	 * Return the list of sharing options.
-	 * @return an array of FreshRSS_Share objects.
+	 * @return array[FreshRSS_Share]
```
Comment (aledeg) [inline app/Models/Share.php:54]: I thought the syntax was ```FreshRSS_Share[]```

Comment (Alkarex) [reply app/Models/Share.php:54]: I think there is more than one syntax, and I am not sure which is most common / best (although I think that `array`  is more usual than `[]`).
Feel free to change.
https://stackoverflow.com/questions/778564/phpdoc-type-hinting-for-array-of-objects/23072048#23072048
https://stackoverflow.com/questions/2713710/comment-associative-array-in-php-documentor/2725073

**Pull request merged**